code/modules/ruins/spaceruin_code/clownstation.dm

```diff
@@ -0,0 +1,56 @@
+
+/obj/item/paper/fluff/ruins/clownstation/mimenote
+	info = "silentio omnes consumentur"
+
+/obj/item/paper/fluff/ruins/clownstation/entry
+	name = "visitor notice"
+	info = "While the Honk Salvation Monastery welcomes pilgrims and visitors alike, please be mindful of the monks who live here and their needs. Additionally, please wear clown religious garb for the duration of your time visiting. If you borrow one of our masks, make sure to return it when you leave."
+
+/obj/item/paper/fluff/ruins/clownstation/honkfather
+	name = "visitor notice"
+	info = "Many visitors ask why we choose to display a statue of the Honkfather. The Honkfather is a deeply controversial figure, oftern thought to represent the dangers of pranking in bad fun and the way pranking can turn into crime. We choose to honor the Honkfather, because although he is imperfect, he is a member of the divine harlequin and ultimately a holy force for good. Let us not forget his actions in the book of Bananacus, where he cured the tribe of Pierrots of their sadness, or his prophesided role in The Great World Peeling, where it is said he will fight by the Honkmother's side with his duel tommy guns. An additional note about our statue- the gun used is a replica and cannot fire."
+
+/obj/item/paper/fluff/ruins/clownstation/essay
+	info = "A brief Dissertation on the metaphysical state of the Spirit of The Honk: The Honkspel of Buttons makes many references to a 'spirit of the honk', but does little to describe it. Scrinkles's Honkspel goes into more detail, describing it as 'a massive clown with flesh of pure bananium which eminated pure and unmuted honking'. While this physical description- the very one the statue in this monastery is based on- is appealing, it is also unsatasfying. The Honk is an eternal energy. It cannot hold a single physical form. It permiates all of life and rings into the depths of every clown's mind. The honk of every individual horn is intrinsically a part of the same eternal honk. For this reason, I do not believe the Spirit of the Honk can be encapsulated in a physical being. Rather, its energy vibrates through and controls matter. The spirit does not so much exist as a giant clown of bananium as it exists around it and within it."
+
+/obj/item/paper/fluff/ruins/clownstation/honkprayer
+	name = "prayer"
+	info = "I honk myself with the sacred honk. It is the same honk that honked the universe into existence. It is the honk that honked the breath of life into the first clown. It is the eternal honk. Amen."
+
+/obj/item/paper/fluff/ruins/clownstation/bananaprayer
+	name = "prayer"
+	info = "Hail the banana tree, full of pranks. The Honkmother is with thee. Blessed art thou amongst plants, and blessed is the fruit of thy branch, the banana. Holy banana tree, Mother of pranks, grow for us prankers, now and at the hour of our death, Amen."

```

Comment:
should "Mother" be capitalized?

Reply:
I've always seen Honkmother written with just the H capitalized, but I could be wrong. I think it's one word, not Honk Mother